src/encoded/static/components/view_controls.js

```diff
@@ -343,18 +363,21 @@ BatchDownloadModal.propTypes = {
     downloadClickHandler: PropTypes.func.isRequired,
     /** Called when the user does something to close the modal */
     closeModalHandler: PropTypes.func.isRequired,
+    /*** Result total */
```

Comment:
Could this be more descriptive? That’s just the name of the property. This also has an ESLint because of the three asterisks.